frontend/src/components/common/checkbox/checked.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="9" height="11" viewBox="0 0 9 11" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M4.29382 8.20027L4.29372 8.20037C4.20775 8.28685 4.09032 8.33521 3.96821 8.33521C3.84609 8.33521 3.72866 8.28685 3.64269 8.20037L3.64259 8.20027L0.759741 5.31696L0.759738 5.31695C0.480069 5.03728 0.480117 4.58383 0.759705 4.30477L0.759742 4.30473L1.12071 3.94367C1.12071 3.94367 1.12071 3.94367 1.12072 3.94366C1.40047 3.66401 1.85338 3.66402 2.13302 3.94367L3.93285 5.74359L3.9682 5.77895L4.00356 5.74359L8.86695 0.880105C9.14672 0.600429 9.60008 0.600476 9.87924 0.880072L9.87926 0.880095L10.2402 1.24116L10.2756 1.20581L10.2402 1.24117C10.5199 1.52083 10.5199 1.97419 10.2403 2.25336L10.2402 2.25338L4.29382 8.20027Z" fill="white" stroke="#F3427F" stroke-width="0.1"/>
+</svg>
```

Comment:
Maybe it would be better to put it in the assets/img or assets/icons

Reply:
Good catch!

Agreed